### ISSUE TITLE
tplist ImportError: invalid class name USDTReader

### ISSUE THREAD

**Issue by rickysarraf** [iovisor/bcc#855]
This is seen on the 0.2.0 version of bpfcc, on top of Debian with kernel 4.8.14.
In bcc module, there only a USDT Class.

```
rs@learner:~$ sudo tplist  
Traceback (most recent call last):
  File "/usr/sbin/tplist", line 16, in <module>
    from bcc import USDTReader
ImportError: cannot import name USDTReader
2016-12-13 / 18:17:31 ♒♒♒  ☹  => 1  
```

**Comment by palmtenor** [comment]
This should be already been fixed in #698 .

**Comment by rickysarraf** [comment]
Yes. Looking at those changes, it looks so. I'm closing this bug report. But it'd be nice to have a new release tagged soon.
